packages/yoastseo/src/languageProcessing/languages/ja/Researcher.js

```diff
@@ -40,6 +42,10 @@ export default class Researcher extends AbstractResearcher {
 		delete this.defaultResearches.getFleschReadingScore;
 		delete this.defaultResearches.getPassiveVoiceResult;
 
+		// Replaces default researches with custom researches.
+		delete this.defaultResearches.wordCountInText;
+		this.defaultResearches.wordCountInText = textLengthResearch;
+
```

Comment:
Unless you had a specific reason for replacing the research this way, maybe you can instead use the Object.assign method at the bottom of the file, for consistency with other researches?

Reply:
Changed as suggested 👍